Taskfile.yml

```diff
@@ -103,6 +103,7 @@ tasks:
     desc: Ensure all code is formatted
     dir: v2
     cmds:
+      - golangci-lint run --fix ./...
```

Comment:
This is called both in the `ci` task and the local task `default`. Do we want it executed in both - is the experience w/ this changing code in CI good?

Reply:
Maybe we should change the `verify-no-changes` task to have an error that gives the cmd to run to fix things up?

Reply:
Good idea. Done.